### DISCUSSION TITLE
Integration tests with Kratos

### DISCUSSION THREAD

**Discussion by Leo843** [ory/kratos#2067] [Q&A] [answered]
What are the recommended approaches to test a UI that uses Kratos?

In my case, I am using cypress to test common authentication use cases. I am looking for a way to add/delete identities from within cypress before running tests. There is the admin endpoint but I could not find any API documentation.

**Comment by aeneasr** [comment, marked as answer]
We don't have official documentation on this, but you can take a look at the project's own UI tests :)

https://github.com/ory/kratos/tree/master/test/e2e/cypress

**Reply by Leo843** [reply]
Thank you! This is definitely what I was looking for.
